linker/shared/src/main/scala/org/scalajs/linker/checker/ClassDefChecker.scala

```diff
@@ -331,9 +331,7 @@ private final class ClassDefChecker(classDef: ClassDef,
     val envJustBeforeSuper = checkBlockStats(body.beforeSuper, startEnv)
     checkTreeOrSpreads(body.superCall.args, envJustBeforeSuper)
     val envJustAfterSuper = envJustBeforeSuper.withThisType(instanceThisType)
-    val afterSuperStoreModulesHandled =
-      handleStoreModulesAfterSuperCtorCall(body.afterSuper)
-    checkBlockStats(afterSuperStoreModulesHandled, envJustAfterSuper)
```

Comment:
As I commented on #5057, I don't think we should do this (but start emitting store module for JS modules as well).